packages/fury-adapter-oas3-parser/lib/parser/oas/parseOauthFlowObject.js

```diff
@@ -0,0 +1,67 @@
+const R = require('ramda');
+const {
+  isExtension, hasKey, getValue,
+} = require('../../predicates');
+const {
+  createInvalidMemberWarning,
+} = require('../annotations');
+const pipeParseResult = require('../../pipeParseResult');
+const parseObject = require('../parseObject');
+const parseString = require('../parseString');
+
+const name = 'Oauth Flow Object';
+const requiredKeys = ['scopes'];
+
+/**
+ * Parse Oauth Flow Object
+ *
+ * @param namespace {Namespace}
+ * @param element {Element}
+ * @returns ParseResult
+ *
+ * @see https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#oauthFlowObject
+ * @private
+ */
+function parseOauthFlowObject(context, object) {
+  const { namespace } = context;
+  const scopesName = `${name}' 'scopes`;
+
+  const parseScopeMember = R.cond([
+    [R.T, parseString(context, scopesName, false)],
+  ]);
+
+  const parseMember = R.cond([
+    [hasKey('scopes'), R.compose(parseObject(context, scopesName, parseScopeMember, [], true), getValue)],
+    [hasKey('refreshUrl'), parseString(context, name, false)],
+    [hasKey('authorizationUrl'), parseString(context, name, false)],
+    [hasKey('tokenUrl'), parseString(context, name, false)],
+
+    // FIXME Support exposing extensions into parse result
+    [isExtension, () => new namespace.elements.ParseResult()],
```

Comment:
This line is missing coverage too, test like https://github.com/apiaryio/api-elements.js/blob/e3fe71b97b804f935ab27e95c0c5850ef2e3d480/packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseParameterObject-test.js#L435 would ensure this behaviour works